app/controllers/MessageController.js

```diff
@@ -0,0 +1,31 @@
+Message = require('../models/Message');
+Chat = require('../models/Chat');
+
+exports.createChat = (req, res) => {
+    if(!req.body.text) {
+        res.status(400).send({
+            message: 'Write something...'
+        });
+        return;
+    }
+    Chat.find({
+        $or:[
+            {
+                first_member: req.body.user_id
+            },
+            {
+                second_member: req.body.user_id
+            }
+        ]
+    }).then(data => {
+        if(data === null) {
```

Comment:
@adilet-reactjs We can't check data for null here
Because we are trying to get the list of chats, so if it's a list - array will be returned
Please use data.length here